Documentation/6.0.2/README.rst

```diff
@@ -514,12 +514,26 @@ This example shows how to create a deployment with one Controller node and two C
 
     ::
 
-        KernelArgs: "hugepages=12831 iommu=pt intel_iommu=on isolcpus=1-7"
+        KernelArgs: "default_hugepagesz=1G hugepagesz=1G hugepages=64 iommu=pt intel_iommu=on isolcpus=1-7"
 
     .. Note:: Above kernel arguments are consumed by the another env file which include in deployment command `/usr/share/openstack-tripleo-heat-templates/environments/host-config-and-reboot.yaml`
 
     .. Note:: You also can set GpgCheck to "no" in environment files if user want to disable GPG Check while installating packages on AVRS Node deployment.
 
+    d. For IsolatedCPU or CPUAffinity to be respected, CPUSET_ENABLE needs to be set to the value 0
+
+    ::
+
+        CpuSetEnable        =====>    CPUSET_ENABLE
+
+    e. For AVRS, FLOW_EVICTION_THRESHOLD and CONN_OBJ_THRESHOLD is now configurable in `/etc/default/openvswitch`. `CONN_OBJ_THRESHOLD` should be half the value configured for `FLOW_EVICTION_THRESHOLD`. For example, `FLOW_EVICTION_THRESHOLD=300000` and `CONN_OBJ_THRESHOLD=150000`.
```

Comment:
For AVRS, FLOW_EVICTION_THRESHOLD and CONN_OBJ_THRESHOLD is now configurable in `/etc/default/openvswitch`. Please contact Nuage for the recommended values for these parameters.

>>>
Please notice these are sample templates and parameter values can be customized depending on the use case. Please contact Nuage for the recommended values for these parameters,